server/views/layout.njk

```diff
@@ -80,9 +80,19 @@
       {% block content %}{% endblock %}
     </main>
 
-    <section class="contact-helpdesk" role="complementary">
-      <a href="" target="_blank" class="govuk-link govuk-link--inverse" data-qa="contact-helpdesk">Contact helpdesk (opens in new tab)</a>
-    </section>
+    {% if pageId not in contactHelpdeskBannerExcludedOnPages %}
```

Comment:
If the page id is not in the array of page ids that are excluded from having the contact helpdesk banner ...